.gitignore

```diff
@@ -1,2 +1,3 @@
+.idea
```

Comment:
Well, we have it in framework https://github.com/yiisoft/yii2/blob/master/.gitignore#L2